src/client/index.ts

```diff
@@ -291,44 +328,32 @@ class XrplClient extends EventEmitter {
    * general use. Instead, use rippled's built-in pagination and make multiple
    * requests as needed.
    */
-  async _requestAll(
-    command: 'account_offers',
-    params: AccountOffersRequest
-  ): Promise<AccountOffersResponse[]>
-  async _requestAll(
-    command: 'book_offers',
-    params: BookOffersRequest
-  ): Promise<BookOffersResponse[]>
-  async _requestAll(
-    command: 'account_lines',
-    params: AccountLinesRequest
-  ): Promise<AccountLinesResponse[]>
-  async _requestAll(
-    command: string,
-    params: any = {},
-    options: {collect?: string} = {}
-  ): Promise<any[]> {
+  // TODO: add all requests here once they're done
+  async _requestAll(request: AccountOffersRequest): Promise<AccountOffersResponse[]>
+  async _requestAll(request: BookOffersRequest): Promise<BookOffersResponse[]>
```

Comment:
It's an incomplete list, this PR isn't quite ready yet because I'm still waiting for the rest of the req/resp PRs to be merged (hence the `TODO` above, when I'll do more of a deep dive).